app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,15 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    
+    <%= stylesheet_link_tag 'application', media: 'all' %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
   </head>
 
```

Comment:
- [ ] After I installed gems, set up the database, install packages, and precompile the assets. I still have an error raised when I run the rails server, Please fix that.

![image](https://github.com/tarikbouari/Hello_rails_react/assets/68796232/e2f68c9b-ec89-4c30-8d01-ec276b30b804)

Reply:
@Omar-Muhamad I don't have any issue running the server at my end. you should check may be your rails environment

Reply:
Sorry for the late reply,
I setup everything before I run the dev server but still have this error.
See this [article](https://www.digitalocean.com/community/tutorials/how-to-set-up-a-ruby-on-rails-v7-project-with-a-react-frontend-on-ubuntu-20-04) may help you fix the problem.
In this article the best way to make a react rails app in one repo.